src/main/java/com/griddynamics/cloud/learning/dal/repositories/Repository.java

```diff
@@ -0,0 +1,87 @@
+package com.griddynamics.cloud.learning.dal.repositories;
+
+import com.griddynamics.cloud.learning.dal.connection.DBConnectionManager;
+import com.griddynamics.cloud.learning.dal.domain.DomainEntity;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Getter
```

Comment:
It's a very bad idea to have any state in such "service" classes, they are supposed to be stateless, therefore let's get rid from these getters and setters